vevestaX/vevesta.py

```diff
@@ -344,6 +344,9 @@ def __EDA(self, fileName):
         if self.__data.empty or len(self.__data)==0:
             return
 
+        if type(self.__data) != pandas.core.frame.DataFrame:
```

Comment:
The EDA will execute when self.__data == pandas.core.frame.DataFrame so that we can write similar check for pyspark...this code doesnt solve the problem.